CONTRIBUTING.md

```diff
@@ -20,6 +20,9 @@
     - [Créer un tampon de signature pour la génération PDF](#créer-un-tampon-de-signature-pour-la-génération-pdf)
```

Comment:
The more human-readeabl, interpreted version [here](https://github.com/MTES-MCT/trackdechets/blob/f0d4e2876ccd564c1bab7b04501c5003bda87e93/CONTRIBUTING.md#d%C3%A9pannage)